### ISSUE TITLE
An appalling lemma discrepancy

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/UD_English-EWT#480]
```
# sent_id = reviews-071650-0012
# text = I was appalled.
1       I       I       PRON    PRP     Case=Nom|Number=Sing|Person=1|PronType=Prs      3       nsubj   3:nsubj _
2       was     be      AUX     VBD     Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin   3       cop     3:cop   _
3       appalled        appalled        ADJ     JJ      Degree=Pos      0       root    0:root  SpaceAfter=No
4       .       .       PUNCT   .       _       3       punct   3:punct _
```

vs

```
# sent_id = newsgroup-groups.google.com_INTPunderground_b2c62e87877e4a22_ENG_20050906_165900-0015
24      I       I       PRON    PRP     Case=Nom|Number=Sing|Person=1|PronType=Prs      26      nsubj:pass      26:nsubj:pass   _
25      am      be      AUX     VBP     Mood=Ind|Number=Sing|Person=1|Tense=Pres|VerbForm=Fin   26      aux:pass        26:aux:pass     _
26      appalled        appal   VERB    VBN     Tense=Past|VerbForm=Part|Voice=Pass     14      ccomp   14:ccomp        _
```

Should this be `appal` instead of `appall`, anyway?  Similar question for `enrol`, I suppose.  It doesn't look right to my American sensibilities.

**Comment by AngledLuffa** [comment]
Thoughts on this?  I'd like to switch it to the double letter versions

**Comment by nschneid** [comment]
Yeah definitely not "appal". I guess it should be a VERB, with lemma "appall".

**Comment by AngledLuffa** [comment]
That's not the only time there's `I was [a-z]+ed` with `ADJ/JJ` as the tag

```
# sent_id = newsgroup-groups.google.com_INTPunderground_b2c62e87877e4a22_ENG_20050906_165900-0048
# text = ... I was shocked at the lack of racial diversity.
30      shocked shocked ADJ     JJ      Degree=Pos      0       root    0:root  _

# sent_id = reviews-126171-0003
# text = ... but I was disappointed with their customer service.
16      disappointed    disappointed    ADJ     JJ      Degree=Pos      4       conj    4:conj:but      _


# sent_id = reviews-360937-0002
# text = I must say, I was impressed with the size ...
7       impressed       impressed       ADJ     JJ      Degree=Pos      3       ccomp   3:ccomp _


# sent_id = email-enronsent12_01-0069
# text = I didn't feel guilty about the garage sale, that's why I was annoyed - being notified at 10:00 at night GRRRRRRR.
16      annoyed annoyed ADJ     JJ      Degree=Pos      13      advcl:relcl     13:advcl:relcl  _

# sent_id = email-enronsent37_01-0105
# text = And I was relieved when Nicki called to let me know she was home safe.
4       relieved        relieved        ADJ     JJ      Degree=Pos      0       root    0:root  _


# sent_id = newsgroup-groups.google.com_herpesnation_c74170a0fcfdc880_ENG_20051125_075200-0011
# text = ... before I was finished being a teenager.
35      finished        finished        ADJ     JJ      Degree=Pos      28      advcl   28:advcl:before _
```


although not consistently so:

```
# sent_id = weblog-blogspot.com_rigorousintuition_20060511134300_ENG_20060511_134300-0249
# text = I was amazed at the spiel they delivered.
3       amazed  amaze   VERB    VBN     Tense=Past|VerbForm=Part|Voice=Pass     0       root    0:root  _
```

It looks like *usually* a state of mind is tagged as `ADJ` here

**Comment by nschneid** [comment]
Deciding VERB (past participle) vs. ADJ is very nuanced. I defer to @amir-zeldes when it comes up. (One test for ADJ is *un-* negation, which works for some of these. Another is *very* modification. I'm not sure there's a good reason that *shocked* should be ADJ but *amazed* should be VERB.)

**Comment by amir-zeldes** [comment]
It is indeed very tricky, since the original PTB guidelines are internally inconsistent/offer contradictory tests without a ranking. I do rely on PTB/ON precendents in doubtful cases, but my basic methodology and hopefully the one you find in practice in the GU corpora is articulated [here](https://github.com/UniversalDependencies/UD_English-EWT/issues/290#issuecomment-1785996363), and I'll repeat it for convenience - the priorities of importance we have noticed in prior corpora and what we enforce is, in order:

1. Negation, a.k.a. "no such verb" - if the item has negation that would preclude a verbal lemma, it must be an adjective (uncredited -> JJ). This outranks even a by-agent; exception: if the "un-" lemma exists as a verb (e.g. Santorini's example "untied" -> "untie"). Similar logic applies to the 'no VBG for no such verb' rule, e.g. "outgoing" cannot be VBG, because "outgo" is not a verb.
2. By-agent - if it has an overt by agent, then it is VBN. Santorini originally ranked this below intensifiers, but the corpora seem to behave the opposite way so I have followed them and I think we've been pretty consistent about it. It's also in the spirit of prioritizing argument structure where possible.
3. If a relative clause paraphrase is possible, prefer VBG/VBN, but if that changes the meaning, prefer JJ: "appetizing dish" ->*dish which appetizes -> JJ; existing safeguards ->safeguards that exist -> VBG
4. If "get" is possible but "become" is bad, prefer VBN - Santorini's "I was/got/*became married"
5. Prefer JJ for state if it has a different reading from the event (e.g. "I was mistaken/JJ" != "someone mistook me")

In OntoNotes, it seems "impressed" is about 50-50 when not used in a perfect construction/as a finite verb, and tagged VBN whenever "by" appears (criterion 2), otherwise JJ. Criterion 4 would justify this behavior IMO, but we have 3. ranked higher, which is why it is always VBN in GUM in this function. Personally I would opt for VBN here, it seems pretty transparent to me.

**Comment by nschneid** [comment]
> ON has both tags for lowercased "united", though JJ is the majority tag. For the criteria in the link, the relative paraphrase decides it for VBN ("united people" can be "people who are united by ...")

_Originally posted by @amir-zeldes in https://github.com/UniversalDependencies/UD_English-GUM/issues/78#issuecomment-1847631652_

If test 3 permits adding a copula, then "united states" => "states that are united" clearly passes, but so would canonical stative adjectives ("tall people" => "people who are tall"). So I'm not sure how that would favor the verb analysis.

If it doesn't permit adding a copula, then "states that united" is not quite the same meaning, though it is related by the inchoative alternation.

**Comment by amir-zeldes** [comment]
> but so would canonical stative adjectives

Yes, this is true, but it's more consistent with VBG (uniting factors -> factors which unite), which can always be construed that way, and I don't see why active participles get to stay verbs in this use when passives don't. For me the question is mainly one of the lexeme (lemma), and note that not all participle-like forms pass this test, for example "missing documents" are not "documents which miss".

**Comment by rueter** [comment]
Hi, @nschneid !
Couldn't this also be understood as partially aspect. 
I'm thinking of the -ing participle vs the -ed participle. So, the -ing participle is imperfect and -ed perfect.
How does the word "unite" fit into this, you ask? Well, maybe, "united" is simply the perfect aspect, such that ongoing is dynamic when completed, the result, is stative.
I would see both -ing and -ed participles as possibly verb-based.

**Comment by amir-zeldes** [comment]
> Well there's a semantic difference—"uniting" is dynamic, "united" is stative. Paraphrasing as a finite verb would usually favor the dynamic reading except with verbs that are inherently stative like "exist".

Right, but even for "existing" if I search for DT followed by it, I get 36:3 VBG... So it seems at least for active participles, PTB recognizes the deverbal nature regardless of lexical stativeness

**Comment by amir-zeldes** [comment]
OK thanks - I wouldn't be surprised if there are also internal inconsistencies within GUM and EWT, it's probably a longer term to do but worth looking at at some point.

**Comment by AngledLuffa** [comment]
Just realizing, looking at the current states of the treebanks, EWT still has `United_ADJ` everywhere and GUM has `United_VERB`.  

Looking through EWT examples, I see there is this:

```
# sent_id = weblog-blogspot.com_alaindewitt_20060924104100_ENG_20060924_104100-0091
it is crucial that the forces on the offensive be united_ADJ
```

whereas GUM has

```
# sent_id = GUM_fiction_oversite-47
but I think about that particular time with my mother because Renata was there and we were united_VERB
```

I don't have any horse in this race, but it'd be nice to have it unified one way or another.

**Comment by nschneid** [comment]
> If a relative clause paraphrase is possible, prefer VBG/VBN, but if that changes the meaning, prefer JJ: "appetizing dish" ->*dish which appetizes -> JJ; existing safeguards ->safeguards that exist -> VBG

@amir-zeldes Do you have some VBN examples for both outcomes of the relative clause test? I am thinking of words like "broken" which clearly can be an ADJ ("a very broken system") but you can say "a system which broke/has broken".

(Incidentally, "dish which appetizes" doesn't sound terrible to me...maybe "appetize" as a verb is nonstandard but it does occur on the web.)

**Comment by amir-zeldes** [comment]
> I don't have any horse in this race, but it'd be nice to have it unified one way or another.

I think we agreed on Unite_VERB?

> Do you have some VBN examples for both outcomes of the relative clause test? 

Sure, you can try making it a passive predicate with aux:pass and obl:agent - if it's bad, it supports JJ. For example:

  * official concerted/JJ action - ?the action was concerted by officials
  * armed/JJ conflict - ?the conflict was armed by the belligerents
  * tenured/JJ professor - ?the professor was tenured by the university

This usually works fine for VBNs like:

  * captured/VBN Union soldiers - the Union soldiers were captured by the confederacy
  * measures of perceived/VBN employee discrimination - the discrimination was perceived by the employees
  * scheduled/VBN flights - the flights were scheduled by the company

**Comment by nschneid** [comment]
> * ?the professor was tenured by the university

This sentence sounds fine to me. "Tenure" as a verb may be recent but is well-attested.

> * official concerted/JJ action - ?the action was concerted by officials
> * armed/JJ conflict - ?the conflict was armed by the belligerents

These are good examples of two reasons that the relative clause paraphrase test argues against the participle:

(1) Historical drift: The word in question, despite looking like a participle, has no transparent synchronic relation to a verb. ("Concerted" does relate to an archaic sense of the verb.)
(2) Argument structure: There is a corresponding verb, but the noun being described would not be its subject (passive subject, in the case of VBN). (An "armed conflict" is a conflict among people who are armed.)

Those exceptions are rather narrow, so it seems like the test will strongly favor the participial interpretation over the adjective interpretation. For "united", I think the *get*/*become* test (number 4) is more in line with my own intuitions ("became united" is more natural -> ADJ).

Lowercase "united" in OntoNotes:

<img width="286" alt="Image" src="https://github.com/user-attachments/assets/aa41697b-47c7-40e3-b2e8-2a16a136cd61" />

Notably 2 of the 8 VBNs are "get united" examples.

Going back to the 3rd test:

> If a relative clause paraphrase is possible, prefer VBG/VBN, but if that changes the meaning, prefer JJ: "appetizing dish" ->*dish which appetizes -> JJ; existing safeguards ->safeguards that exist -> VBG

What is an example where we would need to go to test 4—i.e. the relative clause paraphrase is impossible, and not merely because it changes the meaning? The "married" example would be VBN under test 3 right?

**Comment by AngledLuffa** [comment]
> Yeah, it's clearly not consistent in ON. Are we reopening "united"? I thought that was settled already?

I brought it up again because it's still tagged differently between EWT and GUM

**Comment by amir-zeldes** [comment]
Above @nschneid said:

> I'm willing to change EWT's treatment of "united" to VERB/VBG per GUM policies

So I thought this particular item was settled, if not we can discuss more.

**Comment by nschneid** [comment]
> I Googled it in quotes and the first 10 results for me were all false positives (not part of the same constituent).

Try ["tenured by the university"](https://www.google.com/search?sourceid=navclient&ie=UTF-8&oe=UTF-8&q=%22tenured%20by%20the%20university%22)

**Comment by nschneid** [comment]
> But I think there can still be doubt, for example for "married", the sense distinction is pretty subtle, and a "married couple" is definitely "a couple that has been married". It's just that adding the "by" phrase suggests a different event (they were married by a priest or some official), whereas you could be common-law married or something. I think the get/become test can be helpful in nudging towards JJ if you're on the fence because a "by" phrase and paraphrased participle is possible but there may be a different sub-sense.

So I worry that test 3 devolves into pretty subjective judgments about senses. E.g. I don't think there are two senses of "married"—whether the agent is a judge, or a cleric, or some abstract entity like "the state", these are all about formally bringing about a marriage. In PropBank they're all [marry.01](https://verbs.colorado.edu/propbank/framesets-english-aliases/marry.html).

Can the tests 3-5 be revised so that instances of a drastic meaning shift or synchronically unavailable verb are first designated JJ, and the others are subjected to the "get"/"become" test?

**Comment by nschneid** [comment]
So by "sense" you mean something argument structure–specific? If there is an alternation, e.g. John opened the door / The door opened, you are calling those different senses?

If that's true I don't see why an aspectual difference (like stative vs. inchoative) should not also trigger a different sense. Being currently united (stative) may entail having gone through a process of uniting (inchoative), but it doesn't automatically mean that they should get the same tag in my view.

I don't have a super strong intuition about "united" and of course it would be nice for EWT to be compatible with GUM. But if the Penn annotators of EWT had different criteria for tagging participles in general I want to know the scope of how much review would be needed to make JJ vs. VBN consistent across EWT and GUM. Presumably there are lots of adjectives to reconsider, and until we do that I guess the majority treatment of "united" in OntoNotes is a reasonable proxy for how the rest of EWT is annotated with respect to JJ vs. VBN.

**Comment by amir-zeldes** [comment]
> So by "sense" you mean something argument structure–specific? If there is an alternation, e.g. John opened the door / The door opened, you are calling those different senses?

No, the participants in those are the same. But "marrying someone to someone else" involves officiating a wedding, whereas marrying someone as a personal act can happen via common law, or might not even require an officiator at all in many cultures. To me it feels like the act that a priest performs as part of their vocation is a distinct and more narrow sense than people marrying each other in general. But I think we're paying too much attention to marriage here - I don't think it's a typical example and maybe the sense distinction is borderline/depends on how deault-y the person's reading of the situation involving an officiator is.

> I don't have a super strong intuition about "united" and of course it would be nice for EWT to be compatible with GUM. But if the Penn annotators of EWT had different criteria for tagging participles in general I want to know the scope of how much review would be needed to make JJ vs. VBN consistent across EWT and GUM.

Indeed it would be nice for them to be compatible, and I thought we had agreed on that above. If not then I could reiterate some of the considerations from my perspective, but they're also right here above and in #290. I also like following PTB precedent as you know, but for VBN vs. JJ the PTB is clearly not consistent, and seems to be influenced in practice by some of the same considerations (has the agent been mentioned nearby, is there prefixal un-, etc.). On a more programmatic level, I think making transparent passive participles into adjectives obscures their argument structure and connection to the original verb. We can already tell they are being used adjectivally thanks to the deprel `amod`, but on a morphological level they are still participles. There are cases where that stops being true, such as "concerted" etc. above, but if we can envision a by-agent and there's a clear corresponding verbal lemma, doesn't that mean we are looking at a passive participle?

**Comment by nschneid** [comment]
On a gut level, I guess I would say that if something can be paraphrased as "Something is currently ____", that frame is suggestive of either (a) the progressive construction with a VBG, or (b) a copula construction with an adjective or noun predicate. It is odd to say "The city is currently destroyed by the fire", which could be taken as an indication of morphological deverbalization. "The states are currently united by a common purpose" can be read as a copula construction (there is nothing unusual about adjectives having oblique PP modifiers). Treating that as a progressive construction feels wrong aspectually (even if the semantic roles of the progressive and copula uses are aligned).

That said, I can appreciate that different people will want to draw the line in different places. Given that GUM has articulated a set of guidelines, I'd be happy if someone wanted to apply them to all the EWT JJ's for consistency. Not wild about just making a change for one lexical item in isolation that would create within-corpus inconsistency.

**Comment by amir-zeldes** [comment]
Yeah, I basically don't see aspect as a deciding factor in whether something is morphologically an adjective, for me participles stay participles until something happens to divorce them from their verb (to keep the marriage theme 😉)

I won't say that GUM is 100% consistent about JJ vs. VBN, but whenever some uncertainty has come up in class the criteria above have been what we've invoked, so that's at least the ideal. And if I can err on the side of preserving deverbal semantics and argument structure, then I generally prefer to do that (and in general for morphosyntactic annotation I tend to prefer transparent, regular analyses for as long as possible, until some lexicalization or different sense really breaks the possibility of sticking to that).

**Comment by nschneid** [comment]
I finally had a look at CGEL, which discusses adjectival passives (adjectives derived from past participle verbs) starting on p. 1436. Some points made in this section:

- "Adjectival passives always have a stative interpretation." Verbal passives may be stative or dynamic depending on the active form of the verb (passivizing the verb does not change the aspect).
- In certain contexts there is an ambiguity between adjectival and verbal passives.
- Grammatical tests:
  * modification by *very* is a sufficient condition for adjectival status
  * predicative complements of *seem*, *look*, *remain* etc. are adjectival ("_They seemed very worried_ but not *_The kitchen window seemed broken by the thieves_").
  * *un-* prefixation for negation is specific to adjectives: verbs do not have the same sense of *un-*
- "Adjectival passives may contain a *by* phrase: _He remained too embarrassed by their behavior to acknowledge that he was their son._"

Would the predicative complement test be viable?

scheduled flights => *flights that seem scheduled => VBN
tenured professor => professor that seems tenured => JJ

**Comment by nschneid** [comment]
> For un- negated cases that seems reasonable, because the passive part of the process is hidden on a prelexical level under the derivation. But if the word is functioning as the passive participle of that verb, with the same sense and arguments, shouldn't that be expressed in the morphosyntactic analysis?

I think the question is where to posit derivation. CGEL is saying there is widespread zero-derivation as shown by certain syntactic tests. The presence of a by-phrase is not an absolute test for verb, as is made clear by the un- examples. So it's a question of which tests to prioritize.

Given that the PTB tagset largely aligns with CGEL (modulo "many"/"few" and CGEL's intransitive prepositions—a non-traditional account by their own admission), I am inclined to think that CGEL reflects the conventional wisdom on the adjective/verb distinction within English syntax circles, and could be a way to understand the Penn annotations in EWT.

I agree with you that the predicative complement test may be hard to get sharp judgments for though....

**Comment by amir-zeldes** [comment]
> I think the question is where to posit derivation

Yeah, this might be a key reason underlying my view: I basically think of English participles as an inflectional, not a derivational category, since the form is available for any verb, and is totally predictable for neological verbs. This is not the same as verbal derivation from nouns or the other way around, which is not inflectional.

> CGEL's intransitive prepositions

I think I will never get over this 😄 Historically Germanic prepositions are adverbs that became transitive, not the other way around! Phrasal verbs just retain the older morphological category with some word order restrictions. But I've lost that battle and I'm tagging RP -> ADP all the same...

**Comment by AngledLuffa** [comment]
United might be worth an exception, since United Kingdom and United States are so common
